src/config.js

```diff
@@ -21,7 +21,7 @@
 import browser from 'bowser';
 
 const PREFIX =
-  browser.webkit ? '-webkit-' :
+  browser.webkit || browser.blink ? '-webkit-' :
```

Comment:
Thanks, my point was it would be nice to add this link as a comment in the source code.
```
// See https://github.com/ded/bowser#rendering-engine-flags
```